examples/progress.js

```diff
@@ -0,0 +1,49 @@
+/* @jsx h */
+const {h, mount, Component, Text, Bar} = require('../');
+
+const MAX = 20;
+
+class ProgressApp extends Component {
+	constructor() {
+		super();
+
+		this.state = {
+			i: 0
+		};
+	}
+
+	render() {
+		const text = `Running `;
```

Comment:
I think Ink should handle that for us. That's the behavior I expected. That would mean Ink would provide us the available screen space instead of us using `process.stdout.columns` directly. Let's wait and see what @vadimdemedes thinks.